src/Client.js

```diff
@@ -741,6 +741,41 @@ class Client extends EventEmitter {
         return { gid: createRes.gid, missingParticipants };
     }
 
+    /**
+     * Returns all current Labels
+     * @returns {Promise<Array<Label>>}
+     */
+    async getLabels() {
+        return JSON.parse( await this.pupPage.evaluate(async () => {
```

Comment:
No need to encode/decode JSON here. Puppeteer takes care of serializing the models.
Also, no need to `return await`. Opt for `return` instead. The https://eslint.org/docs/rules/no-return-await rule will be enabled on eslint soon.